commit 3150ac371d56189889d93968ed8b50e72c9e6eb4 (refs/changes/48/3648/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-03-15T10:31:03-07:00 (1 year, 7 months ago)
    
    TRITON-246 TRITON-196 change to machine.deletion_protection missed one test assert update

diff --git a/test/machines/deletion_protection.js b/test/machines/deletion_protection.js
index 7ea71cd..7f67b9a 100644
--- a/test/machines/deletion_protection.js
+++ b/test/machines/deletion_protection.js
@@ -111,7 +111,7 @@ function deletionProtectionTests(suite, client, other, machineUuid, cb) {
             function getCb(err, req, res, body) {
 
             t.ifError(err, 'GetMachine error');
-            t.equal(body.deletion_protection, undefined, 'deletion_protection');
+            t.equal(body.deletion_protection, false, 'deletion_protection');
 
             t.end();
         });
